content/en/security_platform/cloud_workload_security/getting_started.md

```diff
@@ -60,6 +60,60 @@ There are two types of monitoring that the Datadog Agent uses for Cloud Workload
 [2]: https://docs.datadoghq.com/integrations/kubernetes_audit_logs/
 {{% /tab %}}
 
+{{% tab "Kubernetes (DaemonSet)" %}}
+1. If you installed the Datadog Agent as a DaemonSet with a manual [manifest](https://docs.datadoghq.com/agent/kubernetes/?tab=daemonset), then you can add the following lines to activate CWS module :
+
+```
+    containers:
+        - image: 'datadog/agent:latest-jmx'
+          ...
+          securityContext:
+              capabilities:
+                add:
+                  - SYS_ADMIN
+                  - SYS_RESOURCE
+                  - SYS_PTRACE
+                  - NET_ADMIN
+                  - IPC_LOCK
+          env:
+              - name: DD_SYSTEM_PROBE_ENABLED
+                value: "true"
+              - name: DD_RUNTIME_SECURITY_CONFIG_ENABLED
+                value: "true"
+              - name: HOST_ROOT
+                value: "/host/root"
+          volumeMounts:
+              - name: passwd
+                mountPath: /etc/passwd
+                readOnly: true
+              - name: group
+                mountPath: /etc/group
+                readOnly: true
+              - name: root
+                mountPath: /host/root
+                readOnly: true
+              - name: os-release
+                mountPath: /host/etc/os-release
+    ...            
+    volumes:
```

Comment:
Is the indentation correct ?

Reply:
Hello @lebauce fron my point of view yes : I copy-paste the manifest I used in production and strip the non related. Of course someone who wants to use this snippet will have to copy-paste things and respect .yaml indentation. I can add a warning message if you want, let me know

Reply:
Let us know if it's OK for you here @lebauce 👍